_posts/Redis/1300-01-01-Flushall-Redis.md

```diff
@@ -0,0 +1,40 @@
+---
+date: 2018-05-03
+title: Redis's Flushall Is Not Spinnaker's Friend
+categories:
+   - Redis
+description: Why not to run Redis's Flushall with Spinnaker. Flushall. Redis.
+type: Document
+---
+05/03/2018
+
+## Question:
+Should I do a Redis __flushall__ on my Redis database?
```

Comment:
Personal preference: I'd like to use single backticks on FLUSHALL so that it shows up like this `FLUSHALL` since it's a programatic command.